build.gradle

```diff
@@ -18,13 +18,11 @@ buildscript {
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
-    dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.43'
-    }
 }
 
 plugins {
     id 'net.researchgate.release' version '2.3.4'
+    id 'org.jetbrains.intellij' version '0.1.5'
```

Comment:
I think it has something to do with muli-plugin projects like ours. I actually dug up on old issue you were contributing to: https://github.com/JetBrains/gradle-intellij-plugin/issues/17 . My suspicion is that the new version of the plugin is not compatible with the workaround you came up with.